src/client/game.cpp

```diff
@@ -1286,11 +1284,18 @@ void Game::shutdown()
 	FpsControl fps_control;
 	fps_control.reset();
 
+	float percentage = 0;
 	while (stop_thread->isRunning()) {
 		m_rendering_engine->run();
 		f32 dtime;
 		fps_control.limit(device, &dtime);
-		showOverlayMessage(N_("Shutting down..."), dtime, 0, false);
+
+		percentage += 5 * (dtime / 0.1);
+		if (percentage >= 100) {
+			percentage = 0;
+		}
+		
+		m_rendering_engine->draw_load_screen(utf8_to_wide(std::string(N_("Shutting down…"))) , guienv, texture_src, dtime, percentage);
```

Comment:
Why the explicit `std::string`?

Reply:
Ah, actually you broke translations here. This should use `wstrgettext` to match `showOverlayMessage`. (To fix translations and to match the other strings, you should also replace `…` with `...` again.)